### PR TITLE
Merge into master branch

### PR DESCRIPTION
**Pull request by dprado981**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+* The Comment class stores the data for a comment on the webpage
+*/
+package com.google.sps.data;
+
+public final class Comment {
```
Comment (KidEinstein) [inline portfolio/src/main/java/com/google/sps/data/Comment.java:6]: Please add a comment documenting the function of the class.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -36,9 +36,12 @@ <h3>Fun Fact About Me!</h3>
           <td><div id="quote-container"></div></td>
         </tr>
       </table>
+      <hr>
       <form action="/data" method="POST">
-        <p>Comment:</p>
-        <input type="text" name="user-comment" placeholder="Add a comment!">
+        <p>Leave a comment down below!</p>
+        <input type="text" name="user-name" placeholder="Enter your username" required >
+        </br>
```
Comment (KidEinstein) [inline portfolio/src/main/webapp/index.html:43]: Seems like <br> is an empty tag. Why are we closing it here? https://www.w3schools.com/tags/tag_br.asp

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -47,20 +47,65 @@ function changePic(imageContainer) {
...
+  const days = hours / 24;
+  if (days < 31) {
+    return Math.floor(days) + " days ago";
+  }
+  const months = days / 30;
+  if (months < 12) {
+    return Math.floor(months) + " months ago";
+  }
+  return Math.floor(months / 12) + " years ago";
 }
```
Comment (KidEinstein) [inline portfolio/src/main/webapp/script.js:111]: Great idea showing "time ago" instead of a timestamp.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -47,20 +47,65 @@ function changePic(imageContainer) {
...
+  commentElement.appendChild(timeAgoElement);
+  commentElement.appendChild(textElement);
+  
+  return commentElement;
+}
+
+/** returns a phrase describing the difference  
+*   from the current time to when a comment was 
+*   posted, in milliseconds, as a string
+*/
```
Comment (KidEinstein) [inline portfolio/src/main/webapp/script.js:89]: Nit: This comment should start with a capital letter and end with a period.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (dprado981) [reply portfolio/src/main/webapp/index.html:43]: Fixed in latest commit to login branch

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (dprado981) [reply portfolio/src/main/webapp/script.js:89]: Fixed in later commit

**Pull request merged**